src/pages/task/testsTable/StatusSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { useLocation, useHistory } from "react-router-dom";
+import queryString from "query-string";
+import { RequiredQueryParams } from "types/task";
+import { TestStatus } from "types/task";
+import { TreeSelect } from "components/TreeSelect";
+
+export const StatusSelector = () => {
+  const { pathname, search } = useLocation();
+  const { replace } = useHistory();
+  const value = useStatuses(search);
```

Comment:
+1 to abstracting this into a custom hook. only question is about the name -- use statuses how?

Reply:
useStatuses returns the query param statuses